orangecontrib/prototypes/widgets/owsplit.py

```diff
@@ -0,0 +1,136 @@
+import Orange.data
```

Comment:
Change the order of imports -- first Python's, then numpy/scipy/etc, then Qt's, then `orangewidget`, then `Orange`.

Reply:
Oof, I am usually quite careful with that. How did I miss it?